### DISCUSSION TITLE
Host or Node Name(s) in Scheduler Section of /api/v1/health output

### DISCUSSION THREAD

**Discussion by desaibc76** [apache/airflow#32912] [Ideas]
I have been searching for an easy way to find/know where scheduler is running. In our environment, it is running on more than two servers to manage the load and to ensure high availability. Sometimes it is difficult to remember where it is running when you really need it when working on an issue. 

It will be helpful to know if that can be included in output of __**/api/v1/health**__. 

for e.g.
```
{
  "metadatabase": {
    "status": "healthy"
  },
  "scheduler": {
    "latest_scheduler_heartbeat": "2023-07-28T08:55:28.248505+00:00",
    "status": "healthy",
    "node_or_server" : "server1, server4, server10"
  },
  "triggerer": {
    "latest_triggerer_heartbeat": null,
    "status": null
  }
}
```

It is even better if we know which scheduler is unhealthy from the output so that particular node/server can be taken out of rotation for investigation and RCA.

```{
    "metadatabase": {
      "status": "healthy"
    },
    "scheduler": {
        "node_or_server_1" : {
            "latest_scheduler_heartbeat": "2023-07-28T08:55:30.248505+00:00",
            "status": "healthy"
        },
        "node_or_server_2" : {
            "latest_scheduler_heartbeat": "2023-07-28T08:55:31.248505+00:00",
            "status": "un-healthy"
        },
        "node_or_server_3" : {
            "latest_scheduler_heartbeat": "2023-07-28T08:55:31.548505+00:00",
            "status": "healthy"
        },
    },
    "triggerer": {
      "latest_triggerer_heartbeat": null,
      "status": null
    }
}
```

**Comment by potiuk** [comment]
I think it will be better to add it to the cluster activity view that comes in 2.7. https://user-images.githubusercontent.com/14861206/239577995-14a372b1-131f-42bc-b984-54455a15454b.png 

Fel free to raise PR for that.
